src/main/java/homeworks/task_4.java

```diff
@@ -0,0 +1,14 @@
+package homeworks;
+
+public class task_4 {
+    public static void main(String[] args) {
+
+        for (int i = -0; i >= -10; i = i-2) {
```

Comment:
-0? how is it?) 
also you can use here % 
e.g. **i** % 2 == 0 so in this case **i** is a even number